native/explorer/src/dataframe/io.rs

```diff
@@ -47,7 +47,13 @@ pub fn df_from_csv(
         _ => CsvEncoding::Utf8,
     };
 
-    let dataframe = CsvReadOptions::default()
+    let read_options = if dtypes.is_empty() {
+        CsvReadOptions::default()
+    } else {
+        CsvReadOptions::default().with_schema_overwrite(Some(schema_from_dtypes_pairs(dtypes)?))
```

Comment:
Maybe instead of a conditional, we could have `schema_from_dtypes_pairs` return Some/None accordingly, and we always call with_schema_overwrite?

Reply:
Good call! I changed in https://github.com/elixir-explorer/explorer/pull/955/commits/846017001330ccd05852a012c2d08c8607dba70d